### ISSUE TITLE
Crash with basic private method example

### ISSUE THREAD

**Issue by kt3k** [denoland/deno#12940]
Found in https://github.com/denoland/deno_std/pull/1655

The following example exits with error status code at the middle of execution.
```js
class A {
  a() {
    this.#b()
  }
  #b() {}
}
for (let i = 0; i < 1e5; i++) {
  console.log(i);
  new A().a();
}
```

```shellsession
$ deno run test.js
...
10623
10624
10625
10626
10627
10628
10629
zsh: trace trap  deno run test.js
```

Note: It doesn't have even rust backtrace. It always stops at 10629 on my machine (mac).

Deno v1.15.3 doesn't have this issue. It seems to have started from v1.16.0.

**Comment by sno2** [comment]
Can confirm it also shows same behavior and stops at `10629` on Windows and Linux (Ubuntu via WSL):

**Comment by DjDeveloperr** [comment]
Stops on 10225 for me. You can use `echo %ERRORLEVEL%` to know the exit code on Windows, shows `-2147483645` but if you convert it to unsigned then to hex it's `0x80000003` (`STATUS_BREAKPOINT`) 🤔

**Comment by sno2** [comment]
> Stops on 10225 for me. You can use `echo %ERRORLEVEL%` to know the exit code on Windows, shows `-2147483645` but if you convert it to unsigned then to hex it's `0x80000003` (`STATUS_BREAKPOINT`) 🤔

Ah, nice.  I just used `$LastExitCode` for powershell and also got exit code `-2147483645` on Windows.

**Comment by piscisaureus** [comment]
Looks like this is probably a V8 bug.

**Comment by kt3k** [comment]
Reported https://bugs.chromium.org/p/v8/issues/detail?id=12500

**Comment by syg** [comment]
Hi, V8 dev here. It is indeed a V8 bug, thank you for reporting. I'm happy deno is exercising the private field code paths and exposing these issues.

**Comment by lucacasonato** [comment]
Merged into 9.8 in https://chromium-review.googlesource.com/c/v8/v8/+/3386595. The fix will be present in Deno 1.18.

**Comment by bartlomieju** [comment]
Fixed in 1.18
